src/applications/disability-benefits/2346/config/form.js

```diff
@@ -138,14 +138,19 @@ const formConfig = {
   version: 0,
   prefillEnabled: true,
   title: 'Order hearing aid batteries and accessories',
-  finishLaterLinkText: 'Finish this order later.',
   subTitle: 'VA Form 2346A',
   savedFormMessages: {
     notFound:
       'You can’t reorder your items at this time because your items aren’t available for reorder or we can’t find your records in our system. For help, please call the Denver Logistics Center (DLC) at 303-273-6200 or email us at dalc.css@va.gov.',
     noAuth: 'Please sign in again to continue your application for benefits.',
     forbidden:
       'We can’t fulfill an order for this Veteran because they are deceased in our records. If this information is incorrect, please call Veterans Benefits Assistance at 800-827-1000, Monday through Friday, 8:00 a.m. to 9:00 p.m. ET.',
+    reviewPageTitle: 'Review order details',
+    appSavedSuccessfullyMessage: 'Order has been saved.',
+    startNewAppButtonText: 'Start a new order',
+    continueAppButtonText: 'Continue your order',
+    finishAppLaterMessage: 'Finish this order later.',
+    appType: 'order',
```

Comment:
`savedFormMessages.reviewPageTitle` doesn't make much sense, does it? `savedFormMessages.startNewAppButtonText`?

`savedFormMessages` is intended to store messages to show on the page after attempting to save the form.

@Clark-W-Griswold may have opinions on how to group these better, but I think inside `savedFormMessages` isn't the right place.

Reply:
@cvalarida and @Clark-W-Griswold, please let me know what would be a better alternative. I was under the impression that you were okay with grouping the custom messages together.

Reply:
I'd recommend keeping the old `savedFormMessages` and the properties that were in there, and moving all the new things you added into a `customText` property.

Reply:
@cvalarida is that necessary for this PR or a nice to have?